### PR TITLE
feat(docs): add badges to stories

### PR DESCRIPTION
**Pull request by gfellerph**
Adding badges to stories gives users useful meta information about the state of the story.

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: .changeset/spicy-paws-beg.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-documentation': minor
+---
+
+Added badges to stories. Badges give important meta information about the state of a story
```
Comment (alizedebray) [inline .changeset/spicy-paws-beg.md:5]:
```suggestion
Added badges to stories. Badges give important meta information about the state of a story.
```
:wink:

File: packages/documentation/package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
   },
   "devDependencies": {
     "@babel/core": "7.20.12",
+    "@geometricpanda/storybook-addon-badges": "^0.2.2",
```
Comment (alizedebray) [inline packages/documentation/package.json:39]:
```suggestion
    "@geometricpanda/storybook-addon-badges": "0.2.2",
```

File: packages/documentation/src/stories/foundation/accessibility.stories.mdx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import { Meta } from '@storybook/addon-docs';
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
```
Comment (alizedebray) [inline packages/documentation/src/stories/foundation/accessibility.stories.mdx:2]: This import can be removed

File: packages/documentation/src/stories/foundation/layout/layout.stories.mdx [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import { Meta } from '@storybook/addon-docs';
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
```
Comment (alizedebray) [inline packages/documentation/src/stories/foundation/layout/layout.stories.mdx:2]: Same here, you can drop the import

**Review by alizedebray — APPROVED** [review]

**Pull request merged**